webapp/views/wpt-results.js

```diff
@@ -1164,18 +1207,66 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     return `${this.getTestNumbersDisplay(passes, total, isDir)} (${cellDisplay}%)`;
   }
 
+  // Formats the numbers shown on the results page for the test view.
+  formatCellDisplayTestView(passes, total, status, isDir) {
+
+    // At the test level:
+    // 1. Show PASS is passes == total for subtests.
+    // 2. Show FAIL if status is undefined (legacy summaries) or 'O' (because showing OKAY would be misleading).
+    // 3. Show FAIL otherwise.
+    if (!isDir) {
+      if (passes === total) {
+        return "PASS"
+      } else if ((status === undefined) || (status === 'O')) {
+        return "FAIL";
+      } else if (status in STATUS_ABBREVIATIONS) {
+        return STATUS_ABBREVIATIONS[status];
+      } else {
+        return "FAIL";
+      }
+    }
+
+    const formatPercent = parseFloat((passes / total * 100).toFixed(0));
```

Comment:
lt is hard to compare float without loss of precision, in general you should instead check abs(formatPercent - 100.0) < EPSILON, where EPSILON is the precision loss due to the float representation, or use string equality directly, without `parseFloat` from the result of toFixed(0).

Reply:
Good catch. This was actually unused. Removed.